operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2053,6 +2086,326 @@ gateway:
+        maxSurge: 0
+        maxUnavailable: 1
+
+    terminationGracePeriodSeconds: 180
+
+    env: []
+    extraEnvFrom: []
+    tolerations: []
+    podDisruptionBudget: {}
+    initContainers: []
+    extraContainers: []
+    extraVolumes: []
+    extraVolumeMounts: []
+
+  write_proxy:
+    replicas: 2
+
+    service:
+      annotations: {}
+      labels: {}
+
+    # -- Resources for graphite-write-proxy pods
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
+
+    # -- Additional graphite-write-proxy container arguments, e.g. log level (debug, info, warn, error)
+    extraArgs: {}
+    # -- The name of the PriorityClass for graphite-write-proxy pods
+    priorityClassName: null
+    # -- Labels for graphite-write-proxy pods
+    podLabels: {}
+    # -- Annotations for graphite-write-proxy pods
+    podAnnotations: {}
+    # -- Node selector for graphite-write-proxy pods
+    nodeSelector: {}
+    # -- Affinity for graphite-write-proxy pods
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: target
+                    operator: In
+                    values:
+                      - graphite-write-proxy
+              topologyKey: 'kubernetes.io/hostname'
+
+    livenessProbe:
+      httpGet:
+        path: /ready
+        port: http-metrics
+      initialDelaySeconds: 45
+    readinessProbe:
+      httpGet:
+        path: /ready
+        port: http-metrics
+      initialDelaySeconds: 45
+
+    annotations: {}
+    persistence:
+      subPath:
+    securityContext: {}
+    strategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxSurge: 0
+        maxUnavailable: 1
+    # -- Grace period to allow the graphite-write-proxy to shutdown before it is killed
+    terminationGracePeriodSeconds: 180
+    env: []
+    extraEnvFrom: []
+    tolerations: []
+    podDisruptionBudget: {}
+    initContainers: []
+    extraContainers: []
+    extraVolumes: []
+    extraVolumeMounts: []
+
+  web:
+    replicas: 1
+
+    image:
+      repository: docker.io/raintank/graphite-mt
+      tag: 8-7ebde4c60-v2
+      pullPolicy: IfNotPresent
+
+    service:
+      annotations: {}
+      labels: {}
+
+    # -- Resources for graphite-web pods
+    resources:
+      requests:
+        cpu: 100m
+        memory: 512Mi
+    # -- The name of the PriorityClass for graphite-web pods
+    priorityClassName: null
+    # -- Labels for graphite-web pods
+    podLabels: {}
+    # -- Annotations for graphite-web pods
+    podAnnotations: {}
+    # -- Node selector for graphite-web pods
+    nodeSelector: {}
+    # -- Affinity for graphite-web pods
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: target
```

target is an outdated label, so this affinity rule doesn't match anything you can use app.kubernetes.io/component

```suggestion
                  - key: app.kubernetes.io/component
```